docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -214,7 +214,38 @@ For more information about devfile contents requirements, see the "What is outer
 +
 **Solution:** Select a predefined runtime type from the **Review and configure for deployment** view. Make sure you meet all of the {ProductName} <<repository requirements>> for the runtime you select, or configure your own <<custom build and deployment instructions>>.
 
+=== Using registry.redhat.io base images for builds 
+
+Retrieve the Red Hat Container Registry token and configure your application to use this token. Using this token improves the build process of your application. 
+
+.Procedure
+
+Retrieve the Red Hat Container Registry token by following these steps:
+
+. Go to link:https://access.redhat.com/terms-based-registry/#/[Registry service account]. 
+. Create a new Registry service account by selecting *New Service Account*.  
+. Retrieve the Registry service account token as a “Docker Configuration”:
+```
+   {
+    "auths": {
+        "registry.redhat.io": {
+        "auth": "MTEwNzAxOTl8-redacted-c2Jvc2U6ZZlNDQ="
+        }
+    }
+    }
+``` 
+.Procedure 
+
+Configure your application to use the Red Hat Container Registry token by following these steps: 
```

Comment:
ah, ok. i see. so i'd call procedure 1 something like "Retrieving the ... token from _________," and this second one something like "Configuring your application to ...". then the title of the whole section could be "Retrieving and configuring ...". i'm still not clear, though, where i'm retrieving it from or why.